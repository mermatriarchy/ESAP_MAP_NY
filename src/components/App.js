import React, { useState } from 'react';

import NavBar from './NavBar';
import Map from './Map';
import About from './About';
import Footer from './Footer';

export default function App() {
  const [count, setCount] = useState(0);
  return (
    <div>
      <NavBar />
      <Map />
      <About />
      <Footer />
    </div>
  );
}
