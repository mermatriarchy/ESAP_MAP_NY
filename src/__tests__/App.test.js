import React from 'react';
import { shallow } from 'enzyme';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import App from '../components/App';

Enzyme.configure({ adapter: new Adapter()});

it('correctly renders the top-level App component', () => {  
    const AppComponent = shallow(<App />);
    expect(AppComponent).toMatchSnapshot();
});