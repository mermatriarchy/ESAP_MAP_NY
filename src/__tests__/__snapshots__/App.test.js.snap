// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correctly renders the top-level App component 1`] = `ShallowWrapper {}`;
